sale_financial_risk_info/tests/test_sale_financial_risk_info.py

```diff
@@ -5,44 +5,34 @@
 
 
 class TestSaleFinancialRiskInfo(TransactionCase):
-    def setUp(self):
-        super().setUp()
-        self.Partner = self.env["res.partner"]
-        self.Product = self.env["product.product"]
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.Partner = cls.env["res.partner"]
+        cls.Product = cls.env["product.product"]
 
-        self.product_1 = self.Product.create(
+        cls.product_1 = cls.Product.create(
             {"name": "Product test 1", "list_price": 1000.00}
         )
-        self.partner_1 = self.Partner.create(
+        cls.partner_1 = cls.Partner.create(
             {
                 "name": "partner test rebate 1",
                 "ref": "TST-001",
                 "risk_invoice_draft_include": True,
             }
         )
-        self.default_account_revenue = self.env["account.account"].search(
-            [
-                ("company_id", "=", self.env.company.id),
-                (
-                    "user_type_id",
-                    "=",
-                    self.env.ref("account.data_account_type_revenue").id,
-                ),
-            ],
-            limit=1,
-        )
-        self.invoice_partner_1 = self.create_invoice(self.partner_1)
+        cls.invoice_partner_1 = cls.create_invoice(cls.partner_1)
 
     # Create some invoices for partner
-    def create_invoice(self, partner):
+    @classmethod
+    def create_invoice(cls, partner):
         move_form = Form(
-            self.env["account.move"].with_context(default_type="out_invoice")
+            cls.env["account.move"].with_context(default_move_type="out_invoice")
```

Comment:
In v16, it's even desirable to not use `Form` for invoices. Check https://github.com/odoo/odoo/blob/f73c214efdbd49865ff3f0db915fadc80b4403b0/addons/account/tests/common.py#L674